src/components/Contacts/AddContact.js

```diff
@@ -73,7 +76,15 @@ const useForm = (addNewContact, accountId, emailSuggest) => {
     }));
   };
 
-  return [values, handleChange, handleSubmit];
+  const handleCheckBoxChange = event => {
+    event.persist();
+    setValues(values => ({
+      ...values,
+      [event.target.name]: event.target.checked,
```

Comment:
The one thing that is a bit awkward about using "event.target.name" here is that it means the interface that the frontend has with the api is set by the "name" parameter in the UI, which links two things that could otherwise be pretty disparate. I think this anti-pattern also exists in other places in the code base, but we may want to go over and address it at some point. I'm not sure now is the time to do that, though.